conf/variant/rpi-qtauto/local.conf.sample

```diff
@@ -5,3 +5,8 @@ MACHINE = "raspberrypi3"
 RPI_USE_U_BOOT = "1"
 
 LICENSE_FLAGS_WHITELIST = "commercial_faad2"
+
+
+# Improves build time a lot
+# DISTRO_FEATURES_remove = "webengine"
```

Comment:
why is this here?

Reply:
This reduces the build time in the area of hours. Web engine depends on qt and usually, you don't want to rebuild it, so we should mention this somewhere to save developers time.

Reply:
Trust me, we're aware of that :)